scripts/makecrates.py

```diff
@@ -72,21 +71,18 @@
 license = "MIT/Apache-2.0"
 
 [dependencies]
-bare-metal = "1.0.0"
+critical-section = {{ version = "1.0", optional = true }}
+cortex-m = "0.7.6"
+cortex-m-rt = {{ version = ">=0.6.15,<0.8", optional = true }}
```

Comment:
@adamgreig Do we still need this workaround?

Reply:
It doesn't hurt, we are still compatible with cortex-m-rt 0.6.15 and lets people still use it if they want.